.travis.yml

```diff
@@ -1,13 +1,11 @@
 os: linux
-dist: xenial
+dist: focal
 language: python
 cache: pip
 arch: amd64
```

Comment:
I think this can also be removed.

Reply:
@willianrampazzo yes! I just removed it and replaced it with a list of all the supported arches. This made the YAML file shorter :)